batch/airflow/example_dag.py

```diff
@@ -0,0 +1,70 @@
+#
```

Comment:
Is this python script running inside of a container? Do we need a Dockerfile checked in?

Reply:
This is [DAG](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html). Airflow runs it inside of special pod (mount volume with this script which should be placed in GCS bucket). So, no need of Dockerfile.